tests/e2e/support/commands.js

```diff
@@ -91,6 +99,11 @@ Cypress.Commands.add('login', (options = {}, route, isMockingExternalRequests =
   cy.openPopup(async (contentWindow) => {
     /* eslint-disable-next-line no-param-reassign */
     contentWindow.localStorage.state = JSON.stringify(await getLoginState(options));
+
+    if (options.pendingTransaction) {
+      // eslint-disable-next-line no-param-reassign
+      contentWindow.localStorage['sh-wallet_transactions_ae_uat'] = JSON.stringify(preparePendingTransactionToLocalStorage(options.pendingTransaction));
```

Comment:
The key for the local storage should be created out of the constants we have in the system.